packages/tools/viewer-alpha/vite.config.mjs

```diff
@@ -54,6 +54,13 @@ export default defineConfig(({ mode }) => {
             alias: {
                 core: `@${source}/core/dist`,
                 loaders: `@${source}/loaders/dist`,
+                inspector: `@${source}/inspector/dist`,
+                //"shared-ui-components": `@${source}/sharedUiComponents/dist`,

```

Comment:
should be removed ?

Reply:
@ryantrem want to take care of that in your push to the PR?

Reply:
Yes, I'm happy to address any remaining issues in this PR.